packages/rich-text/src/component/use-anchor.js

```diff
@@ -7,6 +7,8 @@ import { useState, useLayoutEffect } from '@wordpress/element';
 /** @typedef {import('../register-format-type').WPFormat} WPFormat */
 /** @typedef {import('../types').RichTextValue} RichTextValue */
 
+const STABLE_OBJECT = {};
```

Comment:
what is this stable object?

Reply:
https://github.com/WordPress/gutenberg/pull/59635#discussion_r1517628025